(ForumsTeam)Broken Link

kadamajay56 opened the pull request:
@jimdial  : There is a broken link for "high availability strategy for network virtual appliances" under User-defined section. I have updated the link with relevant one. Kindly review and do the needful.

azurecla commented:
Hi __@BrucePerlerMS__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (bruceper). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

kadamajay56 committed:
(ForumsTeam)Broken Link

There is a broken link for "high availability strategy for network virtual appliances" under User-defined section. I have updated the link with relevant one. Kindly review and do the needful.
